src/robotoff.ts

```diff
@@ -61,7 +61,19 @@ const robotoff = {
       });
   },
 
-  questions(filterState, count = 10, page = 1) {
+  questions(
+    filterState: {
+      insightType: string;
+      brandFilter: string;
+      valueTag: string;
+      countryFilter: string;
+      sortByPopularity: boolean;
+      campaign: string;
+      predictor: string;
+    },
```

Comment:
I don't remember if we always provide the full filter params object

```suggestion
    filterState: Partial<{
      insightType: string;
      brandFilter: string;
      valueTag: string;
      countryFilter: string;
      sortByPopularity: boolean;
      campaign: string;
      predictor: string;
    }>,
```